File: src/Business/Services/ProjectGroupService.java
package Business.Services;

import Business.Services.Base.*;
import Common.DataAccess.*;
import Common.Models.*;
import Common.Services.*;

public class ProjectGroupService extends ReferenceParentEntityService<ProjectGroup, Project> implements IProjectGroupService {
    public ProjectGroupService(
            IGlobalDataAccess globalDataAccess,
            IProjectGroupDataAccess entityDataAccess,
            IProjectDataAccess childEntityDataAccess) {
        super(globalDataAccess, entityDataAccess, childEntityDataAccess);
    }
}

File: backend/Business.Impl/Services/ProjectGroupService.cs
using Business.Contracts.Services;
using Business.Impl.Services.Base;
using Business.Models.Entities;
using DataAccess.Contracts;
using Shared.Contracts;

namespace Business.Impl.Services;

internal sealed class ProjectGroupService : GroupService<ProjectGroup, Project>, IProjectGroupService
{
	public ProjectGroupService(ISharedContext sharedContext, IAppUnitOfWork unitOfWork)
		: base(sharedContext, unitOfWork, unitOfWork.ProjectGroupRepo, unitOfWork.ProjectRepo)
	{
	}
}
